static/sass/styles.scss

```diff
@@ -323,6 +323,19 @@ p + pre {
 
 // XXX Peter: adding border back for fieldsets
 fieldset {
-  border: 1px solid $color-light;
-  border-radius: $sp-xx-small;
+  border: 1px solid $color-mid-light;
+  border-bottom-width: 0;
+  border-radius: $sp-xx-small $sp-xx-small 0 0;
+}
+
+fieldset + fieldset {
+  border-top-width: 0 !important;
```

Comment:
@steverydz Do we need any of these `!important`s?

Reply:
Yes, it wasn't working without them sadly :(